eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -1249,6 +1249,54 @@ public function testLoadLocationChildrenDataWithOffsetAndLimit(LocationList $loc
         );
     }
 
+    /**
+     * @covers \eZ\Publish\API\Repository\LocationService::loadLocationChildren
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    public function testLoadLocationChildrenRespectsParentSortingClauses(): void
+    {
+        $repository = $this->getRepository();
+        $locationService = $repository->getLocationService();
+
+        $parentLocation = $locationService->loadLocation(5);
```

Comment:
You should not rely on pre-existing data (except for root location) because it might disappear. I think if you create a structure dedicated for this test, it's going to be also more clear that sorting works as we expect.